EG23.tex

```diff
@@ -268,23 +287,32 @@ \subsubsection{Case 1: Greedy Separations}
 the separation rate indirectly using the gradual fuel fabrication capacity
 growth. Full fabrication facilities stop requesting separated plutonium
 leading to eventual halting of spent fuel separations.
-
+As the separation output buffers are not limited, those separation facilities will
+reprocess all the used available fuel with a constant reprocessing rate.
+
+As the fuel fabrication facilities have a tuned deployement and capacities (for 10
+batchs of fuel), the output buffer of the speration facilities have been used as
+input for the ``Plutonium'' data fo the ``Fuel Ressource in storage''. The reprocessed
```

Comment:
What do you mean by "input" here?  I think you mean that the results we are reporting for "Fuel Resource in Storage" (note typo) is based on the inventories of the separation facilities output buffer.